.gitignore

```diff
@@ -1,8 +1,19 @@
+.gitignore
+
 *.iml
 .idea/
 classes/
 
+.project
+.checkstyle
+.classpath
+*.prefs
+
 out/
+bin/
```

Comment:
The [Buildship plugin](https://projects.eclipse.org/projects/tools.buildship) for Eclipse works in this folder. (I can not tell you a lot more, because it is the first time I use Gradle)